lib/scope.js

```diff
@@ -235,6 +235,7 @@ AST_Toplevel.DEFMETHOD("figure_out_scope", function(options){
                     ref.reference(options);
                 });
                 node.thedef = def;
+                node.reference(options);
```

Comment:
> Is `options` ever subsequently referenced by the symbol definition?

No - only `options.keep_fnames` is used to propagate function names for mangle avoidance.